packages/slice-machine/server/src/api/common/error.ts

```diff
@@ -1,4 +1,4 @@
-import type { ApiError } from "@models/server/ApiResult";
+import type { ApiError } from "../../../../lib/models/server/ApiResult";
```

Comment:
module-alises didn't work with rollup + esbuild + typescript.
I tried a few things then figure it would be easier to remove it and re-evaluate what we're doing to support aliases as it's a bit messy having them configured in package.json, tsconfigs, babel and next.

Reply:
Ok, so taking care of the aliases in a next PR.

Reply:
technical debt ticket* with a strategy.

Reply:
Will have to think of a strategy first and see if it meets our requirements.